src/host/polyfills/simpleView.ts

```diff
@@ -92,6 +92,24 @@ export function applyCommonRevealerPatch(content: string) {
     }
 }
 
+export function applyStylesRevealerPatch(content: string) {
+    const pattern = /this\._navigateToSource\(selectElement,\s*true\);/g;
+    if (content.match(pattern)) {
+        return content.replace(pattern, '');
+    } else {
+        return null;
+    }
+}
+
+export function applyLinkActionsPatch(content: string) {
+    const pattern = /\"Elements panel\"/g;

```

Comment:
we should be more specific with this CL, I can see a lot of hits.

Reply:
Removed this function - not needed for most recent iteration.